README.md

```diff
@@ -704,6 +704,25 @@ Only Odoo's:
 
     docker-compose run --rm odoo odoo -u addon1,addon2 --stop-after-init
 
+##### Update changed addons only
+
+Add `module_auto_update` from https://github.com/OCA/server-tools to your
+installation following the standard methods of `repos.yaml` + `addons.yaml`.
+
+Now we will install the addon:
+
+    docker-compose up -d
+    docker-compose run --rm odoo --stop-after-init -u base
+    docker-compose run --rm odoo --stop-after-init -i module_auto_update
+    docker-compose restart odoo
+
+It will automatically update addons that got updated every night.
+To force that automatic update now in a separate container:
+
+    docker-compose up -d
+    docker-compose run --rm odoo autoupdate
+    docker-compose restart odoo
```

Comment:
I can elaborate on this problem in our production instances. In our case, we run 4 Odoo containers - web, website, cron, longpolling.

We're using a strategy very similar to this one at the moment, but originally just started with another container running the `--update=all` command. 

We have found that we need to restart all of the other Odoo instances after the update container stops, regardless of the Python code that is residing in the other Odoo containers. The main thing that seems to be screwed up is all of the XML caching and minimization.